src/map/navi.cpp

```diff
@@ -203,7 +181,8 @@ bool navi_path_search(struct navi_walkpath_data *wpd, const struct navi_pos *fro
 	tp[i].flag = SET_OPEN;
 	tpused[i] = 1 + (from->x << 16 | from->y);
 
-	heap_push_node(&g_open_set, &tp[i]); // Put start node to 'open' set
+	g_open_set.push_back(&tp[i]);
```

Comment:
we should use push_heap here too

Reply:
this is really needed? g_open_set are cleared at this point, push_heap will change [0] to [0]?

Reply:
can you help with replacing the similar macros in timer.cpp? I was unable to successfully apply it there, if it is changed there we can remove the BHEAP macros from the emulator, discord: aoshinho